configs/roles/integrations.json

```diff
@@ -0,0 +1,16 @@
+{
+  "roles": [
+    {
+      "name": "Integrations administrator",
+      "description": "Perform any available operation against any Integrations resource.",
+      "system": true,
+      "platform_default": false,
```

Comment:
The "Notifications administrator" role is a platform default. Integrations does not need to be?

Reply:
Correct.
- Integrations are supposed to be created only by account admins (for now it's a all or nothing)
- Any user can define his/her notifications